spark-bigquery-parent/pom.xml

```diff
@@ -238,6 +239,18 @@
                 <version>${openlineage-spark.version}</version>
                 <scope>test</scope>
             </dependency>
+            <dependency>
```

Comment:
why do we need these dependencies?

Reply:
OpenTelemetry needs to be provided by the environment, as here we have just the API but not the reporting library. If we don't specify it as provided, it will be either (a) shaded, and therefor won't work with the reporter; or (b) unshaded, potentially colliding with the environment's OpenTelemetry. The best option is to avoid including it.

Reply:
Got it. From offline discussion: needed by BQ storage library.